libraries/_autoload.rb

```diff
@@ -1,17 +1,21 @@
-DNSMADEEASY_GEM_VERSION ||= '= 0.3.2'.freeze
 
-begin
-  gem 'dnsmadeeasy', DNSMADEEASY_GEM_VERSION
-rescue LoadError
-  unless defined?(ChefSpec)
-    run_context = Chef::RunContext.new(Chef::Node.new, {}, Chef::EventDispatch::Dispatcher.new)
+def load_gem(gem_name, version)
+  begin
+    gem gem_name, version
+  rescue LoadError
+    unless defined?(ChefSpec)
```

Comment:
Curious why we need a special case for ChefSpec?

Reply:
Remember — this was pulled from the Docker cookbook. We decided that this is needed so that ChefSpec finds the gems. Nevertheless CI is failing because it's not finding them, so I am looking into it.